apollo-biz/src/test/resources/data.sql

```diff
@@ -13,6 +13,46 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
+ALTER TABLE APP ALTER COLUMN DataChange_CreatedBy VARCHAR(255) NULL;
```

Comment:
How about also move these alter statements to apollo-biz/src/test/resources/import.sql?